packages/customize-widgets/src/utils.js

```diff
@@ -102,6 +102,10 @@ export function widgetToBlock( { id, idBase, number, instance } ) {
 		...rest
 	} = instance;
 
+	if ( ! raw.content ) {
+		raw.content = '';
+	}
```

Comment:
it's worth pointing out here that we're directly mutating the original instance object. this may be "safe" because we assert that object is broken, but we may not want to be or intentionally be doing this.

that is, if calling code passes in a an instance and expects it be untouched by this routine it will be surprised because this messes with the input. could be useful here to perform a non-mutating update by creating a new variable to capture the raw content.

```js
const rawContent = raw.content || '';

…

instance: { encoded, hash, raw: { ...raw, content: rawContent }, ...rest }
```

Reply:
That's a very good point, thanks for pointing that out. As you said, it might not matter here because the object is inherently broken, but it's just a bad practice to mutate like this.